include/abb_librws/rws_poco_client.h

```diff
@@ -246,16 +247,16 @@ class POCOClient
    *
    * \param ip_address for the remote server's IP address.
    * \param port for the remote server's port.
-   * \param user for the remote server's authentication (assumed to be Digest).
```

Comment:
This is no longer only Digest auth, or has the default changed?

Reply:
No, the default is still Digest for RWS. However, the POCOClient implementation don't know that it is used for RWS at all, and since the [HTTPCredentials](https://pocoproject.org/docs/Poco.Net.HTTPCredentials.html) can handle both it is more general this way (compared to the previous [HTTPDigestCredentials](https://pocoproject.org/docs/Poco.Net.HTTPDigestCredentials.html)).